transport-native-epoll/src/main/java/io/netty/channel/epoll/TunTapChannelOption.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2014 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.channel.epoll;
+
+import io.netty.channel.ChannelOption;
+
+public final class TunTapChannelOption<T> extends ChannelOption<T> {
+
+    public static final ChannelOption<Integer> READ_BUF_SIZE = valueOf("READ_BUF_SIZE");
+
+    @SuppressWarnings({ "deprecation" })
+    private TunTapChannelOption(String name) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private constructor 'io.netty.channel.epoll.TunTapChannelOption(String)' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
can we just have a default no argument constructor like `EpollChannelOption` and `KQueueChannelOption`?